internal/core/software_renderer.rs

```diff
@@ -1640,21 +1640,20 @@ impl<'a, T: ProcessScene> crate::item_rendering::ItemRenderer for SceneBuilder<'
 
 /// This is a minimal adapter for a Window that doesn't have any other feature than rendering
 /// using the software renderer.
-///
-/// The [`MAX_BUFFER_AGE`](SoftwareRenderer#max_buffer_age) generic parameter is forwarded to
-/// the [`SoftwareRenderer`]
-pub struct MinimalSoftwareWindow<const MAX_BUFFER_AGE: usize> {
+pub struct MinimalSoftwareWindow {
     window: Window,
-    renderer: SoftwareRenderer<MAX_BUFFER_AGE>,
+    renderer: SoftwareRenderer,
     needs_redraw: Cell<bool>,
 }
 
-impl<const MAX_BUFFER_AGE: usize> MinimalSoftwareWindow<MAX_BUFFER_AGE> {
+impl MinimalSoftwareWindow {
     /// Instantiate a new MinimalWindowAdaptor
-    pub fn new() -> Rc<Self> {
+    ///
+    /// The `repaint_buffer_type` parameter specify what kind of buffer are passed to the [`SoftwareRenderer`]
```

Comment:
Looking at this documentation, maybe RepaintBufferKind is a better name?

Reply:
I prefer "Type" (despite Apple's preference on "Kind"). I think Type is simpler english.